packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -117,8 +117,8 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     }
 
     /// @notice Semantic version.
-    /// @custom:semver 3.4.0
-    string public constant version = "3.4.0";
+    /// @custom:semver 3.5.0
+    string public constant version = "3.5.0";
```

Comment:
I assume we aren't using strict semver but just calling out we did break the interface with provenWithdrawals

Reply:
https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/STYLE_GUIDE.md#L93-L97

Reply:
Agreed with clabby, we already have a pending major version update for fpac

Reply:
I think the argument would be that the v3 interface/security model has been on `develop` for some time, and someone might have pulled a hypr and deployed it already. 

OTOH, our README now indicates that contracts should be taken from a governance approved tagged release, which this code has not been included in. So I'm inclined to agree with just keeping v3. 

I concede, resolving.

Reply:
I do think we should consider a way for semver to signal whether something is a prerelease or not. Prereleases are the canonical way to `batch` major updates

Reply:
If using prerelease the version that hits mainnet first is 4.0.0 instead of 4.x.x. all releases hitting mainnet rather than some minor releases being testnet only is another nice to have